README.md

```diff
@@ -26,6 +28,34 @@ To start pi-blaster and have it relaunched automatically on every reboot:
 
     sudo make install
 
+### Build a deb package and install with dpkg
+
+You may prefer to package pi-blaster first to better integrate with Debian system.
+
+#### Raspbian 7 ("Wheezy") users
+
+This is the most common case. If you are not sure which version of Debian you are
+using, follow this instruction.
+
+    sudo apt-get install debhelper dh-autoreconf dpkg-dev autoconf
+    cd /path/to/pi-blaster/
+    dpkg-buildpackage -us -uc -i
+    sudo dpkg -i ../pi-blaster*.deb
+
+#### Raspbian 8 ("Jessie") users
+
+If you have upgraded your Raspberry Pi to use Raspbian 8 with systemd, you may
+still follow the instruction given for Wheezy users. However, an extra
+special step may help you to better integrate with systemd.
+
+    sudo apt-get install debhelper dh-autoreconf dh-systemd dpkg-dev \
+         init-system-helpers autoconf
+    cd /path/to/pi-blaster/
+    git revert 656e14c6acbb77aec95a57783726c043ee642f34
```

Comment:
Why is this needed? This is a pretty convoluted way of switching between with and without systemd. Maybe an easier way would be to just duplicate the debian files in debian7 and debian8 and symlink before building? 
Does Debian provide a standard way to handle this?

I am worried that relying on a git revert is going to seriously confuse people.